insonmnia/node/tasks.go

```diff
@@ -49,28 +49,6 @@ func (t *tasksAPI) List(ctx context.Context, req *pb.TaskListRequest) (*pb.TaskL
 	return reply, nil
 }
 
-func (t *tasksAPI) getSupplierTasks(ctx context.Context, tasks map[string]*pb.TaskStatusReply, deal *pb.Deal) {
```

Comment:
Out of scope.

Reply:
Nope, I've succesfully refactored getWorkerClientByEthAddr usage